posts/2023-12-21_end_of__year__up.../end_of__year__update_from_the__pharmaverse__council.qmd

```diff
@@ -0,0 +1,52 @@
+
+## What We Accomplished
+
+One of our biggest moves in 2023 was to form our partnership with [PHUSE](https://phuse.global/PHUSE_Open_Source_Technology). This latched us into an existing community with shared values and a platform that helps us continue to build the pharmaverse community. At the [PHUSE EU Connect](https://www.phuse-events.org/attend/frontend/reg/thome.csp?pageID=11403&eventID=19&traceRedir=2) we had our first opportunity to host a pharmaverse meetup and bring together pharmaverse contributors in person. As we move forward, we'll continue to use this platform to find ways we can host events and encourage collaboration within the pharmaverse community.
+
+This year our community was also able to launch new platforms to share updates and knowledge throughout the industry. The [pharmaverse examples](https://pharmaverse.github.io/examples/) webpage was launched to show pharmaverse packages in action, and the [pharmaverse blog](https://pharmaverse.github.io/blog/) (which I'm using right here!) provides a platform to share updates and community news. 
+
+## Where Next? 
+
+Back in October, I had the opportunity to do an interview with Michael Rimler for the [PHUSE video series](https://youtu.be/F3vehIoJgdA?si=80p0pKobZqjlXChj) Open Source Technologies in Clinical Data Analytics for open-source technologies in clinical data analytics. The last question he asked me was what I expect the state of data analytics in life science to be in 2 to 3 years. My response was that when that time comes, I hope I couldn't have predicted where we would be - because back in 2020 I could never have predicted where we are now. The progress we've made is unbelievable, and the pharmaverse community has played a huge role in getting us where we are today. For the pharmaverse community, I hope to see that progress continue as we move into next year. As a council, our goal is to continue to mature this community. How can we support and foster collaboration between our organizations? How can we leverage this platform to drive the industry forward?  
+
+For you as an individual, there's always an opportunity to get involved - and you don't have to be a package developer to contribute. You can [join a working group](https://pharmaverse.org/contribute/wg/), write [examples](https://github.com/pharmaverse/examples), or author a [blog post](https://github.com/pharmaverse/blog). Furthermore, you can get started with the pharmaverse packages, provide feedback via issues, and advocate for their use within your own organization. The pharmaverse community doesn't exist without you, and we're happy to have you all here to help us build this together.
+
+Here's to a 2024 full of progress and collaboration! 
```

Comment:
Possibly add a short p.s. to let people know of the PHUSE US Keynote to come in Feb 2024? Good to raise awareness of this in different ways